css/style.css

```diff
@@ -500,6 +981,157 @@
     background-color: #e51937;
     color: #fff;
   }
+
+  .wrap-information {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+    background-position-y: top;
+    height: 1125px;
+    margin-top: 0;
+  }
+
+  .info__title {
+    text-align: right;
+    padding: 0 130px 0 0;
+    margin-top: 45px;
+  }
+
+  .red-first {
+    max-width: 1023px;
+    height: 530px;
+    float: none;
+    padding: 63px 47px 0 0;
+  }
+
+  .redrow {
+    flex-direction: row;
+    display: flex;
+    height: 300px;
+    padding-bottom: 10px;
+  }
+
+  .description, 
+  .description-car {
+    padding-left: 61px;
+  }
+
+  .description__title {
+    font-size: 30px;
+  }
+
+  .drive {
+    max-width: 63%;
+    margin-left: 0;
+  }
+
+  .icons {
+    padding-right: 57px;
+  }
+
+  .data__wireless-update {
+    white-space: nowrap;
+  }
+
+  .wrap {
+    display: flex;
+    justify-content: flex-end;
+    flex-direction: column;
+    height: 1125px;
+  }
+
+  .details {
+    max-width: 1023px;
+    height: 530px;
+    padding: 66px 47px 0 0;
+    margin-top: 50px;
+  }
+
+  .redrow-scnd {
+    flex-direction: row;
+    display: flex;
+  }
+
+  .description-car {
+    margin: 0;
+  }
+
+  .title {
+    padding-top: 0;
+  }
+
+  .description__title-most {
+    font-size: 30px;
+  }
+
+  .model-s {
+    max-width: 63%;
+    margin-left: 0;
+    font-size: 16px;
+    padding-right: 43px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)